translate_srt/__main__.py

```
@@ -0,0 +1,6 @@
+import asyncio
+import sys
```

Ruff (0.8.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)